src/java/org/apache/cassandra/cql3/functions/types/TypeCodec.java

```diff
@@ -17,6 +17,13 @@
  */
 package org.apache.cassandra.cql3.functions.types;
 
+import com.google.common.io.ByteStreams;
+import com.google.common.reflect.TypeToken;
+import org.apache.cassandra.cql3.functions.types.exceptions.InvalidTypeException;
+import org.apache.cassandra.cql3.functions.types.utils.Bytes;
+import org.apache.cassandra.transport.ProtocolVersion;
+import org.apache.cassandra.utils.vint.VIntCoding;
```

Comment:
nit: Do these imports need to move down?

Reply:
No, it looks like the style for IntelliJ from here https://cassandra.apache.org/_/development/code_style.html#imports doesn't specify the correct import order.

Reply:
Updated my style and reformatted them. Now it just fixes one alphabetical issue.